geopandas/base.py

```diff
@@ -790,6 +790,41 @@ def cx(self):
         """
         return _CoordinateIndexer(self)
 
+    def equals(self, other):
+        """
+        Test whether two objects contain the same elements.
+
+        This function allows two GeoSeries or GeoDataFrames to be compared
+        against each other to see if they have the same shape and elements.
+        Missing values in the same location are considered equal. The column
+        headers do not need to have the same type, but the elements within the
+        columns must be the same dtype.
+
+        Parameters
+        ----------
+        other : GeoSeries or GeoDataFrame
+            The other GeoSeries or GeoDataFrame to be compared with the first.
+
+        Returns
+        -------
+        bool
+            True if all elements are the same in both objects, False
+            otherwise.
+
+        Notes
+        -----
+        This function requires that the elements have the same dtype as their
+        respective elements in the other Series or DataFrame. However, the
```

Comment:
```suggestion
        respective elements in the other GeoSeries or GeoDataFrame. However, the
```

We duplicate the information here. The contents of this note is repeating the last sentence of the paragraph above just with more words. Can we either leave out the sentence from above and keep `Notes` or use these two sentences from notes instead of that one above?

Reply:
Yeah, I just copied the docstring from pandas mostly. But you're fully correct, will clean-up a bit.